instrumentation/instasarama/README.md

```diff
@@ -123,8 +123,9 @@ in the message handler to continue the trace.
 
 ### Working With Kafka Header Formats
 
-Starting at v1.2.0, the instrumentation can handle both Kafka headers as binary (legacy) or as string.
-By default, and for versions prior to v1.2.0 , the default behavior is to process Kafka headers in the binary format.
+Starting at v1.2.0, the instrumentation can handle both Instana's trace correlation headers in the binary (legacy) format or in the newer string format.
```

Comment:
`Suggestion`: Since v1.2.0, the instrumentation supports Instana's trace correlation headers in both binary (legacy) and string (new) formats.

Reply:
Thanks, updated.